playwright-e2e/pages/events/create-case/CreateCaseCheckYourAnswersPage.ts

```diff
@@ -5,18 +5,24 @@ import { YesNoRadioEnum } from "../../helpers/enums/RadioEnums";
 export class CreateCaseCheckYourAnswersPage extends BaseJourneyPage{
 
     private readonly applicantInRefugeAnswer: Locator;
+    private readonly respondentInRefugeAnswer: Locator;
     private readonly netAssetsAnswer: Locator;
 
     public constructor(page: Page){
         super(page)
         this.applicantInRefugeAnswer = page.getByRole('row', { name: 'Is the Applicant currently a' }).locator('span').nth(1);
+        this.respondentInRefugeAnswer = page.getByRole('row', { name: 'Is the Respondent currently a' }).locator('span').nth(1);
```

Comment:
Is the Applicant currently a resident in a refuge?

Reply:
I think this is what Playwright Recorder gives you, as the shortest string that matches your locator.  
Up to you @dawudgovuk.